src/openrct2/actions/ClearAction.hpp

```diff
@@ -143,7 +143,7 @@ DEFINE_GAME_ACTION(ClearAction, GAME_COMMAND_CLEAR_SCENERY, GameActionResult)
         do
         {
             tileEdited = false;
-            tileElement = map_get_first_element_at(x, y);
+            tileElement = map_get_first_element_at(TileCoordsXY{ x, y }.ToCoordsXY());
```

Comment:
For future this function `ClearSceneryFromTile` should be refactored to take a `CoordsXY` everything is just dividing and multiplying the x and y values.

Reply:
#10432